src/api/index.js

```diff
@@ -0,0 +1,272 @@
+import config from '../config'
+import {mergeRecursive} from "../helpers/mergeObjects";
+import redirect from "react-router/es/Redirect";
+
+
+class ApiClass {
+    constructor() {
+        this.url = `${config.apiUrl}`;
+    }
+
+    makeRequest = ({url, data, token}) => {
+        const _data = {
+            headers: {
+                'Accept': 'application/json',
+                'Content-Type': 'application/json',
+                ...!!token && {Authorization: `Bearer ${token}`},
+
+            },
+            credentials: 'include',
+
+        };
+
+        const mergedData = mergeRecursive(data, _data);
```

Comment:
I assume that this _data object we should use in all request 
If so - please take a look of using axios interceptor